README.md

```diff
@@ -2,7 +2,10 @@
 
 ## Installation
 
-Copy the extension to `phpBB/ext/phpbb/skeleton`
+To install the source code from this repository, clone this extension repository to `phpBB/ext/phpbb/skeleton`:
+
+	$ git clone https://github.com/phpbb-extensions/phpbb-ext-skeleton.git
```

Comment:
In the contrary with above instruction, this will clone the repo into ```phpbb-ext-skeleton``` folder by default. probably should state destination folder like ```ext/phpbb/skeleton``` etc.
Also, it would state what directory the command should be ran from to clone to the proper place.